pkg/cmd/server/start/node_args.go

```diff
@@ -42,6 +44,7 @@ func BindNodeArgs(args *NodeArgs, flags *pflag.FlagSet, prefix string) {
 	flags.StringVar(&args.VolumeDir, prefix+"volume-dir", "openshift.local.volumes", "The volume storage directory.")
 	// TODO rename this node-name and recommend hostname -f
 	flags.StringVar(&args.NodeName, prefix+"hostname", args.NodeName, "The hostname to identify this node with the master.")
+	flags.StringVar(&args.NetworkPluginName, prefix+"network-plugin", args.NetworkPluginName, "The network plugin to be called for configuring networking for pods.")
```

Comment:
Do we really need to expose this argument here? The number of CLI args is something that we are generally trying to limit in favor of just having the configuration specified in the configuration file.